holidays/utils.py

```diff
@@ -428,7 +428,7 @@ def list_supported_financial(include_aliases: bool = True) -> dict[str, list[str
 
     Args:
         include_aliases:
-            Whether to include entity aliases (e.g. TAR for ECB, XNYS for NYSE).
+            Whether to include entity aliases (e.g. TAR for XECB, NYSE for XNYS).
```

Comment:
Maybe this one too, considering that we're trying to sort alphabetically everywhere 👀
```suggestion
            Whether to include entity aliases (e.g. NYSE for XNYS, TAR for XECB).
```